features/i.yml

```
@@ -1,7 +1,6 @@
-draft_date: 2024-06-05
 name: <i>
-description: The `<i>` element TODO.
+description: The `<i>` HTML element makes text italic in order to identify idiomatic content.
```

Optionally, we could align all of these to use the same pattern, like this:

```suggestion
description: The `<i>` HTML element identifies idiomatic content, styling text as italic by default.
```